SwiftCompilerSources/Sources/Optimizer/PassManager/Context.swift

```diff
@@ -279,6 +279,23 @@ struct SimplifyContext : MutatingContext {
   let preserveDebugInfo: Bool
 }
 
+extension SimplifyContext {
+  func getStaticSize(for type: Type) -> Int? {
+      let v = _bridged.getStaticSize(type.bridged)
```

Comment:
It's better to use -1 as the optional none value. For example, the empty tuple will have a valid 0 size/stride.